### ISSUE TITLE
registerCursorTracker should return an "unregister" function.

### ISSUE THREAD

**Issue by chowey** [nosir/cleave-zen#3]
If I want to use `registerCursorTracker` in a React component, I want to be able to unregister it too.

The function signature should look like this

```ts
type Destructor = () => void;

export declare const registerCursorTracker:
  ({ input, delimiter, delimiters, prefix, }: RegisterCursorTrackerPropsType) => Destructor;
```

Then I can use it in a React component like this:

```tsx
import React from "react";
import { registerCursorTracker } from 'cleave-zen'

const MyComponent: React.FC = () => {
  const ref = React.useRef<HTMLInputElement>();

  React.useEffect(() => {
    if (ref.current) {
      return registerCursorTracker({ input: ref.current });
    }
  }, []);

  return <input ref={ref} />
}
```

With the above, the `registerCursorTracker` will unregister when the component is disposed of.

**Comment by nosir** [comment]
This makes senses to me, will add this.

**Comment by nosir** [comment]
This has been released in `v0.0.17`
